src/test/resources/features/acceptance/decision.feature

```diff
@@ -12,6 +12,13 @@ Feature: Confirm application
     And my entitlement is £9.30 per week with a first payment of £37.20
     And my claim is sent to the back end
 
+  @RequiresWiremock
+  Scenario: Double click on the send application button
+    Given I have entered my details up to the check answers page
+    When I double click send application button in terms and conditions page
```

Comment:
How about `When I double the click send application button on the terms and conditions page`?

Reply:
changed as per comment @mikeSmith08